sdk/contentsafety/azure-ai-contentsafety/README.md

```diff
@@ -119,18 +122,18 @@ ContentSafetyClient contentSafetyClient = new ContentSafetyClientBuilder()
 
 AnalyzeTextResult response = contentSafetyClient.analyzeText(new AnalyzeTextOptions("This is text example"));
 
-System.out.println("Hate severity: " + response.getHateResult().getSeverity());
-System.out.println("SelfHarm severity: " + response.getSelfHarmResult().getSeverity());
-System.out.println("Sexual severity: " + response.getSexualResult().getSeverity());
-System.out.println("Violence severity: " + response.getViolenceResult().getSeverity());
+System.out.println("Hate severity: " + response.getCategoriesAnalysis().get(0).getSeverity());
+System.out.println("SelfHarm severity: " + response.getCategoriesAnalysis().get(1).getSeverity());
+System.out.println("Sexual severity: " + response.getCategoriesAnalysis().get(2).getSeverity());
+System.out.println("Violence severity: " + response.getCategoriesAnalysis().get(3).getSeverity());
```

Comment:
Is it always 1st item be Hate, 2nd item be SelfHarm, etc.?

This is a bit strange. Should the code instead check `getCategory` for category?

Reply:
If customer don't specify the categories, it will be in the order of [Hate, SelfHarm, Sexual , Violence ]
If customer specify the categories, it will outputs the results in the order of request,

Reply:
Then this code may not be a good advice for user. Maybe you should loop the items and print "category" and "severity" instead.

Though not high priority.

Reply:
sure, will do foreach loop in samples